src/explorook-store.ts

```diff
@@ -0,0 +1,21 @@
+import Store = require("electron-store");
+
+export class ExplorookStore extends Store {
+    constructor(name: string = "explorook") {
+        super({
+            name
+        });
+    }
+
+    public getOrCreate(key: string, value: any, onCreated: () => void = null): any {
```

Comment:
we dont need `update`?

Reply:
no, I just extended the store with this helper method (I found myself needing this alot)